api/sixtyfps-cpp/include/sixtyfps.h

```diff
@@ -337,24 +337,67 @@ class Window
     private_api::WindowRc inner;
 };
 
+#if !defined(DOXYGEN)
```

Comment:
This is quite unfortunate. I see that we have the same problem for other enums.
Too bad there is no equivalent to #[doc(inline)] for doxygen.
Can we think of another way?

Reply:
Yeah, it's a bit of a shame. We're almost there since cbindgen extracts the comments as well and doxygen could process them (if generated in time).

Perhaps this is fixable by instructing cbindgen to emit the enum into the public namespace. I can give it a try (separate from this merge request, as this affects other enums as well).